mlir/lib/Conversion/MathToLibm/MathToLibm.cpp

```diff
@@ -185,8 +186,15 @@ void mlir::populateMathToLibmConversionPatterns(RewritePatternSet &patterns) {
   populatePatternsForOp<math::Log10Op>(patterns, ctx, "log10f", "log10");
   populatePatternsForOp<math::Log1pOp>(patterns, ctx, "log1pf", "log1p");
   populatePatternsForOp<math::PowFOp>(patterns, ctx, "powf", "pow");
-  populatePatternsForOp<math::RoundEvenOp>(patterns, ctx, "roundevenf",
-                                           "roundeven");
+  if (options.allowC23Features)
```

Comment:
Could there be a better name for this? Not every user of the math dialect is a C frontend.

Reply:
Right, as also suggested by the `flang` issue raised earlier. So maybe an `allow-roundeven` flag? I'm not keen on adding a flag this specific, but if nothing else in MathToLibm appears to need a flag, that could just be it.